core/src/io/anuke/mindustry/ui/dialogs/LoadoutDialog.java

```diff
@@ -1,16 +1,23 @@
 package io.anuke.mindustry.ui.dialogs;
 
+import io.anuke.arc.collection.Array;
 import io.anuke.arc.scene.ui.TextButton;
 import io.anuke.mindustry.type.*;
 
+import java.util.function.Function;
```

Comment:
Android doesn't support anything out of `java.util.function`, use `io.anuke.arc.function` instead.